--- index.js	2020-03-24 00:41:02.000000000 +0100
+++ label-index.js	2020-04-23 09:32:53.000000000 +0200
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React, { PureComponent } from 'react';
-import { Animated } from 'react-native';
+import { Animated, Text } from 'react-native';
 
 import styles from './styles';
 
@@ -43,7 +43,7 @@
       y1: PropTypes.number,
     }),
 
-    style: Animated.Text.propTypes.style,
+    style: Text.propTypes.style,
     label: PropTypes.string,
   };
 
